packages/angular/projects/ui-angular/src/lib/primitives/checkbox/checkbox.component.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { CheckboxComponent } from './checkbox.component';
+
+describe('CheckboxComponent', () => {
+  let fixture: CheckboxComponent;
+
+  beforeEach(() => {
+    fixture = new CheckboxComponent();
+  });
+  it('Default checkbox behaviour -> ', () => {
```

Comment:
When we write our test names, the idea is that the hierarchy of the `describe` and `it` blocks will provide a narrative and the intent of the test being ran. With that in mind suggest a name similar to `'has a default isChecked value of false'`

Reply:
👍